proto/v1/project_service.proto

```diff
@@ -112,6 +115,51 @@ service ProjectService {
       body: "config"
     };
   }
+
+  rpc ListWebhooks(ListWebhooksRequest) returns (ListWebhooksResponse) {
+    option (google.api.http) = {
+      get: "/v1/{parent=projects/*}/webhooks"
+    };
+    option (google.api.method_signature) = "parent";
+  }
+
+  rpc GetWebhook(GetWebhookRequest) returns (Webhook) {
```

Comment:
We don't need List. Get/Add/Remove/Modify, all as custom method. Also use suffix ":modify"   etc following data source model

Reply:
Only kept the `Add/Update/Remove/Test` method.